pkgs/applications/misc/spacefm/default.nix

```diff
@@ -26,6 +26,11 @@ stdenv.mkDerivation rec {
     ln -s /etc/spacefm/spacefm.conf $out/etc/spacefm/spacefm.conf
   '';
 
+  postFixup = ''
+      wrapProgram $out/bin/spacefm \
```

Comment:
`wrapGAppsHook` already wraps the executable, please use

https://github.com/NixOS/nixpkgs/blob/532bd714bac656e38c3ef5f6d447efb9ffd6792d/pkgs/applications/graphics/gthumb/default.nix#L26-L28